db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+users = User.create!([{email: "mail4@test.pl", first_name: "Kermit", last_name: "Frog",
+ nickname: "kermitFrog", password: "kermitkermit"},
+ {email: "mail1@test.pl", first_name: "Kulfon", last_name: "Maly",
+ nickname: "kulfonMaly", password: "kulfonkulfon"},
+ {email: "mail2@test.pl", first_name: "Muppet", last_name: "Show",
+ nickname: "muppetShow", password: "muppetmuppet"},
+ {email: "mail3@test.pl", first_name: "Maruda", last_name: "Smerf",
+ nickname: "smerfMaruda", password: "smerfsmerf"}])
```

Comment:
I think that code would be more readable and DRY if you'd split data into table of hashes. Then you could just iterate through that data and create new users. There is no need for hash to contain password and email. What do you think about it?